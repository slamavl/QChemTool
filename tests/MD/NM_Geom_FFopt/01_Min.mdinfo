

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1530       1.6628E+03     9.9432E-05     3.6438E-04     F148       62

 BOND    =      344.9387  ANGLE   =      239.4538  DIHED      =      674.1519
 VDWAALS =      286.6516  EEL     =       12.6044  HBOND      =        0.0000
 1-4 VDW =      240.7262  1-4 EEL =     -135.7686  RESTRAINT  =        0.0000
