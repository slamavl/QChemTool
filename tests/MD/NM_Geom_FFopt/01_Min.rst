FGR 
  330
 -10.5865300  -5.2343450   0.2774924 -10.7283640  -5.2650975   1.6215104
 -11.7392961  -5.3064060  -0.4271211  -9.8263842  -6.5509503  -0.2778494
 -10.4652009  -7.5132508   0.4267312  -9.9238974  -6.6583838  -1.6218718
  -9.7534312  -3.9348092  -0.2198681  -9.8366606  -3.9364500  -1.6822873
 -10.5656606  -2.6273831   0.2850643 -10.6805258  -2.6005321   1.6280973
 -11.7324089  -2.6683164  -0.3933350  -9.7561826  -1.3128797  -0.2100682
  -9.8563611  -1.2928385  -1.6709091 -10.5641298  -0.0029506   0.3107051
 -10.6663769   0.0369368   1.6544317 -11.7380742  -0.0378519  -0.3552663
  -9.7565801   1.3086304  -0.1930146  -9.8650796   1.3252980  -1.6530393
 -10.5555613   2.6278918   0.3284374 -10.6458293   2.6822769   1.6726775
 -11.7372280   2.5969244  -0.3243195  -9.7516596   3.9320170  -0.2009419
  -9.9034000   3.9250159  -1.6571695 -10.5489465   5.2478940   0.3561940
 -10.5783811   5.3609893   1.7017060 -11.7671818   5.2555103  -0.2276287
  -9.8192344   6.5117867  -0.3561175  -9.9319046   6.4807364  -1.7016288
 -10.4349358   7.5630037   0.2277137  -8.2810045   6.4792405   0.2010076
  -7.5535429   7.8274983  -0.3283728  -7.6457842   7.8784850  -1.6726121
  -8.1175452   8.8663378   0.3243910  -8.3508006   6.6141480   1.6572361
  -7.5200448   5.2003060  -0.2307468  -7.5646135   5.2219944  -1.6901556
  -8.2635875   3.9124605   0.2308032  -8.3046454   3.9402089   1.6902123
  -7.5251331   2.6071342  -0.2123719  -7.5642532   2.6138815  -1.6714776
  -8.2683736   1.3046322   0.2359710  -8.3183131   1.3128694   1.6950932
  -7.5270742  -0.0013030  -0.2112890  -7.5590336   0.0020413  -1.6704306
  -8.2708763  -1.3088021   0.2288578  -8.3284839  -1.3084726   1.6876972
  -7.5271124  -2.6115519  -0.2173812  -7.5558548  -2.6165689  -1.6767358
  -8.2715455  -3.9173853   0.2221438  -8.3415929  -3.9286120   1.6806384
  -7.5283626  -5.2046137  -0.2224170  -7.5730703  -5.2596404  -1.6809137
  -8.2844170  -6.4792532   0.2195549  -8.3274934  -6.5505332   1.6819719
  -7.5582641  -7.8363758  -0.2853774  -7.5924074  -7.9492573  -1.6284131
  -8.1771138  -8.8263480   0.3929903  -6.0151450  -7.7926182   0.2097983
  -6.0479187  -7.8894162   1.6706369  -5.2846832  -9.1472840  -0.3109737
  -5.3012282  -9.2557576  -1.6547022  -5.9019065 -10.1465034   0.3549674
  -3.7450591  -9.1037348   0.1927885  -3.7849143  -9.2060510   1.6528108
  -3.0020326 -10.4553026  -0.3286611  -3.0000335 -10.5606520  -1.6729025
  -3.6197094 -11.4631758   0.3240666  -1.4706897 -10.4111845   0.2007591
  -1.5526621 -10.5391130   1.6569829  -0.7297465 -11.7595926  -0.3563744
  -0.6464869 -11.8416155  -1.7018854  -1.3322912 -12.8184259   0.2274200
   0.7296547 -11.7596098   0.3559740   0.6463944 -11.8416758   1.7014823
   1.3321913 -12.8184289  -0.2278547   1.4706083 -10.4111893  -0.2011158
   3.0019513 -10.4553368   0.3283014   2.9999523 -10.5607309   1.6725394
   3.6196198 -11.4631932  -0.3244602   1.5525786 -10.5390704  -1.6573439
   0.7434977  -9.1127116   0.2306371   0.7399588  -9.1621726   1.6900452
  -0.7435688  -9.1126982  -0.2309506  -0.7400302  -9.1621104  -1.6903603
  -1.5047888  -7.8205147   0.2122219  -1.5185436  -7.8577870   1.6713268
  -3.0043970  -7.8129111  -0.2361601  -3.0221954  -7.8602583  -1.6952834
  -3.7647221  -6.5179596   0.2110976  -3.7778444  -6.5473298   1.6702384
  -5.2689387  -6.5083435  -0.2290893  -5.2974187  -6.5583765  -1.6879302
  -6.0252731  -5.2128508   0.2171483  -6.0440283  -5.2352553   1.6765021
  -5.2732423  -3.9127741  -0.2196325  -5.2891241  -3.9267360  -1.6788905
  -6.0252090  -2.6103310   0.2194389  -6.0452794  -2.6171261   1.6786963
  -5.2732882  -1.3057225  -0.2170621  -5.2842702  -1.3074771  -1.6762683
  -6.0239955  -0.0013311   0.2216297  -6.0397590  -0.0017022   1.6808893
  -5.2719762   1.3033505  -0.2168794  -5.2843195   1.3056513  -1.6760853
  -6.0216559   2.6079521   0.2201483  -6.0346870   2.6155075   1.6794533
  -5.2693515   3.9109727  -0.2201097  -5.2824183   3.9184851  -1.6794146
  -6.0203690   5.2134358   0.2124189  -6.0457633   5.2439371   1.6715249
  -5.2639821   6.5083472  -0.2359252  -5.2960939   6.5474816  -1.6950470
  -6.0115351   7.7951775   0.1930684  -6.0802090   7.8808043   1.6530937
  -5.2794402   9.1503219  -0.3106537  -5.3651151   9.2189302  -1.6543796
  -5.8361748  10.1844413   0.3553233  -3.7410318   9.1055707   0.2101094
  -3.0073698  10.4638458  -0.2850261  -3.0880628  10.5498987  -1.6280586
  -3.5552828  11.4947497   0.3933774  -3.8084675   9.1823056   1.6709508
  -3.0019226   7.8172139  -0.2288231  -3.0310194   7.8669417  -1.6876623
  -3.7623566   6.5193180   0.2113257  -3.7812237   6.5453204   1.6704674
  -3.0108055   5.2176227  -0.2216003  -3.0183736   5.2314636  -1.6808598
  -3.7646910   3.9140189   0.2169096  -3.7728471   3.9235543   1.6761155
  -3.0126735   2.6093978  -0.2185766  -3.0164409   2.6126929  -1.6777620
  -3.7661303   1.3043816   0.2185990  -3.7708601   1.3059924   1.6777845
  -3.0138566  -0.0005450  -0.2179112  -3.0177678  -0.0003648  -1.6770771
  -3.7673581  -1.3055786   0.2186409  -3.7732552  -1.3081039   1.6778235
  -3.0143580  -2.6098117  -0.2187563  -3.0194561  -2.6136346  -1.6779391
  -3.7674692  -3.9139141   0.2169080  -3.7745179  -3.9225687   1.6761139
  -3.0131855  -5.2162409  -0.2217829  -3.0213508  -5.2296859  -1.6810429
  -1.5072998  -5.2173319   0.2167653  -1.5115168  -5.2291925   1.6759709
  -0.7523204  -6.5188741  -0.2202612  -0.7522558  -6.5339169  -1.6795664
   0.7522695  -6.5188875   0.2200350   0.7522049  -6.5339800   1.6793397
   1.5047276  -7.8205192  -0.2124924   1.5184812  -7.8577423  -1.6715986
   3.0043362  -7.8129425   0.2358886   3.0221353  -7.8603395   1.6950102
   3.7449877  -9.1037575  -0.1931044   3.7848399  -9.2060251  -1.6531302
   5.2846123  -9.1473358   0.3106535   5.3011583  -9.2558553   1.6543782
   5.9018268 -10.1465373  -0.3553226   6.0150838  -7.7926579  -0.2100739
   7.5582036  -7.8364448   0.2850967   7.5923485  -7.9493737   1.6281283
   8.1770442  -8.8263980  -0.3933070   6.0478538  -7.8894060  -1.6709159
   5.2688882  -6.5083926   0.2288593   5.2973696  -6.5584766   1.6876984
   3.7646709  -6.5179817  -0.2113258   3.7777914  -6.5473018  -1.6704677
   3.0131449  -5.2162721   0.2216002   3.0213110  -5.2297677   1.6808597
   1.5072590  -5.2173362  -0.2169472   1.5114752  -5.2291466  -1.6761532
   0.7534320  -3.9137597   0.2185386   0.7543648  -3.9186910   1.6777240
  -0.7534625  -3.9137463  -0.2186753  -0.7543953  -3.9186269  -1.6778608
  -1.5074263  -2.6097952   0.2178346  -1.5092632  -2.6132938   1.6770003
  -0.7536031  -1.3050326  -0.2182041  -0.7545444  -1.3061258  -1.6773591
  -1.5069985  -0.0001146   0.2181657  -1.5084530  -0.0004046   1.6773207
  -0.7533909   1.3051628  -0.2181584  -0.7538741   1.3065716  -1.6773134
  -1.5064348   2.6103600   0.2179260  -1.5085391   2.6136532   1.6770918
  -0.7529853   3.9154235  -0.2186265  -0.7537540   3.9217970  -1.6778091
  -1.5058134   5.2196741   0.2170834  -1.5097771   5.2300588   1.6762897
  -0.7519417   6.5231559  -0.2194188  -0.7560992   6.5439379  -1.6786761
  -1.5018237   7.8244579   0.2174075  -1.5118419   7.8518557   1.6767622
  -0.7431474   9.1220673  -0.2221197  -0.7684551   9.1883455  -1.6806141
  -1.4689883  10.4141351   0.2198973  -1.5091734  10.4870333   1.6823168
  -0.7600986  11.7853824  -0.2774663  -0.8043886  11.9235914  -1.6214840
  -1.2740635  12.8197414   0.4271501   0.7601905  11.7853675   0.2778682
   0.8044815  11.9235323   1.6218905   1.2741636  12.8197454  -0.4267145
   1.4690698  10.4141309  -0.2195398   3.0074512  10.4638129   0.2853868
   3.0881438  10.5498203   1.6284223   3.5553728  11.4947350  -0.3929819
   1.5092565  10.4870771  -1.6819568   0.7432186   9.1220541   0.2224340
   0.7685267   9.1882832   1.6809306   1.5018849   7.8244536  -0.2171365
   1.5119042   7.8519006  -1.6764902   0.7519926   6.5231425   0.2196454
   0.7561501   6.5438745   1.6789034   1.5058542   5.2196698  -0.2169012
   1.5098186   5.2301048  -1.6761072   0.7530159   3.9154100   0.2187633
   0.7537845   3.9217328   1.6779461   1.5064552   2.6103559  -0.2178345
   1.5085598   2.6137000  -1.6770002   0.7534011   1.3051493   0.2182041
   0.7538843   1.3065069   1.6773591   1.5069985  -0.0001187  -0.2181658
   1.5084530  -0.0003576  -1.6773208   0.7535930  -1.3050462   0.2181583
   0.7545343  -1.3061904   1.6773134   1.5074058  -2.6097994  -0.2179260
   1.5092424  -2.6132471  -1.6770919   3.0143377  -2.6098429   0.2186644
   3.0194362  -2.6137170   1.6778470   3.7674385  -3.9139359  -0.2170460
   3.7744860  -3.9225396  -1.6762522   5.2732120  -3.9128230   0.2194933
   5.2890948  -3.9268367   1.6787508   6.0252320  -5.2128902  -0.2173343
   6.0439850  -5.2352436  -1.6766888   7.5283224  -5.2046804   0.2222290
   7.5730316  -5.2597595   1.6807236   8.2843659  -6.4793101  -0.2197897
   8.3274386  -6.5505387  -1.6822092   9.8263337  -6.5510370   0.2776084
   9.9238491  -6.6585200   1.6216266  10.4651413  -7.5133169  -0.4270086
  10.5864885  -5.2344175  -0.2776873  10.7283193  -5.2651223  -1.6217068
  11.7392557  -5.3065131   0.4269210   9.7534008  -3.9348934   0.2197224
  10.5656398  -2.6274550  -0.2851628  10.6805037  -2.6005559  -1.6281950
  11.7323886  -2.6684223   0.3932337   9.8366322  -3.9365886   1.6821414
   8.2715147  -3.9174419  -0.2222868   8.3415599  -3.9286169  -1.6807819
   7.5270923  -2.6116185   0.2172863   7.5558357  -2.6166885   1.6766407
   6.0251885  -2.6103703  -0.2195326   6.0452578  -2.6171137  -1.6787902
   5.2732781  -1.3057714   0.2170154   5.2842605  -1.3075782   1.6762216
   3.7673478  -1.3056003  -0.2186871   3.7732446  -1.3080743  -1.6778698
   3.0138566  -0.0005763   0.2179112   3.0177679  -0.0004475   1.6770770
   3.7661405   1.3043599  -0.2185529   3.7708707   1.3060221  -1.6777384
   3.0126938   2.6093666   0.2186684   3.0164608   2.6126104   1.6778540
   3.7647217   3.9139970  -0.2167716   3.7728788   3.9235834  -1.6759772
   3.0108460   5.2175915   0.2217832   3.0184135   5.2313816   1.6810432
   3.7624077   6.5192958  -0.2110971   3.7812765   6.5453483  -1.6702380
   3.0019834   7.8171826   0.2290952   3.0310796   7.8668605   1.6879360
   3.7411031   9.1055486  -0.2097929   3.8085416   9.1823322  -1.6706316
   5.2795111   9.1502700   0.3109745   5.3651846   9.2188316   1.6547029
   5.8362547  10.1844079  -0.3549662   6.0115963   7.7951372  -0.1927924
   7.5536033   7.8274277   0.3286532   7.6458426   7.8783662   1.6728946
   8.1176149   8.8662859  -0.3240728   6.0802736   7.8808139  -1.6528147
   5.2640326   6.5082979   0.2361553   5.2961429   6.5473811   1.6952785
   6.0204099   5.2133962  -0.2122330   6.0458063   5.2439486  -1.6713379
   5.2693818   3.9109238   0.2202488   5.2824476   3.9183843   1.6795539
   6.0216764   2.6079130  -0.2200550   6.0347086   2.6155201  -1.6793597
   5.2719863   1.3033016   0.2169259   5.2843294   1.3055502   1.6761318
   6.0239955  -0.0013702  -0.2216299   6.0397589  -0.0016893  -1.6808895
   7.5270743  -0.0013694   0.2112888   7.5590337   0.0019216   1.6704305
   8.2708660  -1.3088583  -0.2289059   8.3284728  -1.3084765  -1.6877453
   9.7561725  -1.3129636   0.2100192   9.8563519  -1.2929772   1.6708608
  10.5641298  -0.0030215  -0.3107055  10.6663773   0.0369143  -1.6544306
  11.7380740  -0.0379565   0.3552646   9.7565903   1.3085471   0.1930630
   9.8650893   1.3251599   1.6530883   8.2683840   1.3045763  -0.2359235
   8.3183241   1.3128658  -1.6950453   7.5251534   2.6070677   0.2124663
   7.5642725   2.6137620   1.6715723   8.2636184   3.9124043  -0.2306608
   8.3046784   3.9402048  -1.6900689   7.5200850   5.2002390   0.2309344
   7.5646519   5.2218749   1.6903440   8.2810554   6.4791829  -0.2007731
   8.3508555   6.6141415  -1.6569967   9.8192845   6.5116972   0.3563565
   9.9319516   6.4805971   1.7018670  10.4349952   7.5629306  -0.2274352
  10.5489882   5.2478246  -0.3559993  10.5784265   5.3609685  -1.7015072
  11.7672223   5.2554102   0.2278262   9.7516900   3.9319335   0.2010870
   9.9034285   3.9248778   1.6573146  10.5555822   2.6278215  -0.3283394
  11.7372479   2.5968209   0.3244176  10.6458521   2.6822549  -1.6725774
