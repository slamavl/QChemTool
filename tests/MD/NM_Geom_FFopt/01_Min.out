
          -------------------------------------------------------
          Amber 14 SANDER                              2014
          -------------------------------------------------------

| Run on 03/15/2018 at 21:34:24

|   Executable path: sander
| Working directory: /mnt/sda2/Dropbox/PhD/Programy/Python/QChemTool/tests/MD/NM_Geom_FFopt
|          Hostname: Unknown
  [-O]verwriting output

File Assignments:
|  MDIN: 01_Min.in                                                             
| MDOUT: 01_Min.out                                                            
|INPCRD: nab_input.inpcrd                                                      
|  PARM: nab_input.prmtop                                                      
|RESTRT: 01_Min.rst                                                            
|  REFC: refc                                                                  
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: mden                                                                  
| MDCRD: 01_Min.mdcrd                                                          
|MDINFO: 01_Min.mdinfo                                                         
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

minimize                                                                       
 &cntrl                                                                        
  imin=1,                                                                      
  ntb=0,                                                                       
  ntx=1,                                                                       
  irest=0,                                                                     
  maxcyc=10000,                                                                
  ncyc=4000,                                                                   
  ntpr=100,                                                                    
  ntwx=10,                                                                     
  cut=999.0,                                                                   
 /                                                                             

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
|    NONPERIODIC   ntb=0 and igb=0: Setting up nonperiodic simulation
|  *** cutoff > system size, list only builds once
|Largest sphere to fit in unit cell has radius =    31.728
| New format PARM file being parsed.
| Version =    1.000 Date = 03/15/18 Time = 21:34:24
 NATOM  =     330 NTYPES =       2 NBONH =       0 MBONA  =     390
 NTHETH =       0 MTHETA =     900 NPHIH =       0 MPHIA  =    3126
 NHPARM =       0 NPARM  =       0 NNB   =    2998 NRES   =       1
 NBONA  =     390 NTHETA =     900 NPHIA =    3126 NUMBND =       5
 NUMANG =      10 NPTRA  =       8 NATYP =       4 NPHB   =       0
 IFBOX  =       0 NMXRS  =     330 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0


|     Memory Use     Allocated
|     Real               30567
|     Hollerith            993
|     Integer           174685
|     Max Pairs          54285
|     nblistReal          3960
|     nblist Int       1528485
|       Total             7138 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

FGR                                                                             

General flags:
     imin    =       1, nmropt  =       0

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     100, ntrx    =       1, ntwr    =     500
     iwrap   =       0, ntwx    =      10, ntwv    =       0, ntwe    =       0
     ioutfm  =       0, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       0, igb     =       0, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       0

Energy minimization:
     maxcyc  =   10000, ncyc    =    4000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010
|  INFO: Old style inpcrd file read


--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

FGR                                                                             
 begin time read from input coords =     0.000 ps

 Number of triangulated 3-point waters found:        0

     Sum of charges from parm topology file =  -0.00000002
     Forcing neutrality...

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------

 ---------------------------------------------------

     eedmeth=4: Setting switch to one everywhere

 ---------------------------------------------------
| Local SIZE OF NONBOND LIST =      51288
| TOTAL SIZE OF NONBOND LIST =      51288


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1       1.8321E+03     1.9542E+01     7.5975E+01     C7         19

 BOND    =      443.0070  ANGLE   =      309.4482  DIHED      =      686.2505
 VDWAALS =      290.9881  EEL     =       13.3706  HBOND      =        0.0000
 1-4 VDW =      226.0755  1-4 EEL =     -137.0145  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    100       1.6628E+03     1.6845E-01     5.6390E-01     C26        65

 BOND    =      344.8121  ANGLE   =      239.0422  DIHED      =      674.6359
 VDWAALS =      286.7009  EEL     =       12.6203  HBOND      =        0.0000
 1-4 VDW =      240.8223  1-4 EEL =     -135.7962  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    200       1.6628E+03     2.8628E-02     1.5433E-01     C28        70

 BOND    =      344.7568  ANGLE   =      239.5285  DIHED      =      674.2770
 VDWAALS =      286.5549  EEL     =       12.6020  HBOND      =        0.0000
 1-4 VDW =      240.8124  1-4 EEL =     -135.7643  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    300       1.6628E+03     7.0914E-03     2.3196E-02     C24        60

 BOND    =      344.8773  ANGLE   =      239.4959  DIHED      =      674.2092
 VDWAALS =      286.5577  EEL     =       12.6018  HBOND      =        0.0000
 1-4 VDW =      240.7854  1-4 EEL =     -135.7651  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    400       1.6628E+03     8.6441E-03     2.9492E-02     C30        76

 BOND    =      344.9241  ANGLE   =      239.4652  DIHED      =      674.1872
 VDWAALS =      286.5783  EEL     =       12.6024  HBOND      =        0.0000
 1-4 VDW =      240.7694  1-4 EEL =     -135.7663  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500       1.6628E+03     6.7954E-03     2.2210E-02     C138      305

 BOND    =      344.9134  ANGLE   =      239.4825  DIHED      =      674.1760
 VDWAALS =      286.5963  EEL     =       12.6029  HBOND      =        0.0000
 1-4 VDW =      240.7548  1-4 EEL =     -135.7666  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    600       1.6628E+03     6.1719E-03     2.5767E-02     C28        70

 BOND    =      344.9258  ANGLE   =      239.4687  DIHED      =      674.1696
 VDWAALS =      286.6114  EEL     =       12.6032  HBOND      =        0.0000
 1-4 VDW =      240.7469  1-4 EEL =     -135.7670  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    700       1.6628E+03     1.6619E-03     4.9283E-03     C25        63

 BOND    =      344.9302  ANGLE   =      239.4637  DIHED      =      674.1643
 VDWAALS =      286.6227  EEL     =       12.6036  HBOND      =        0.0000
 1-4 VDW =      240.7414  1-4 EEL =     -135.7676  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    800       1.6628E+03     1.6667E-03     4.7560E-03     C64       149

 BOND    =      344.9297  ANGLE   =      239.4644  DIHED      =      674.1607
 VDWAALS =      286.6309  EEL     =       12.6038  HBOND      =        0.0000
 1-4 VDW =      240.7366  1-4 EEL =     -135.7678  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    900       1.6628E+03     1.5529E-03     4.7138E-03     C138      305

 BOND    =      344.9357  ANGLE   =      239.4572  DIHED      =      674.1581
 VDWAALS =      286.6371  EEL     =       12.6040  HBOND      =        0.0000
 1-4 VDW =      240.7339  1-4 EEL =     -135.7681  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000       1.6628E+03     1.4550E-03     4.7656E-03     C5         14

 BOND    =      344.9347  ANGLE   =      239.4585  DIHED      =      674.1561
 VDWAALS =      286.6415  EEL     =       12.6041  HBOND      =        0.0000
 1-4 VDW =      240.7314  1-4 EEL =     -135.7682  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1100       1.6628E+03     1.1520E-03     4.2747E-03     C29        73

 BOND    =      344.9386  ANGLE   =      239.4539  DIHED      =      674.1548
 VDWAALS =      286.6448  EEL     =       12.6042  HBOND      =        0.0000
 1-4 VDW =      240.7300  1-4 EEL =     -135.7684  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1200       1.6628E+03     1.0917E-03     5.6441E-03     C29        73

 BOND    =      344.9376  ANGLE   =      239.4551  DIHED      =      674.1537
 VDWAALS =      286.6472  EEL     =       12.6043  HBOND      =        0.0000
 1-4 VDW =      240.7286  1-4 EEL =     -135.7685  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1300       1.6628E+03     1.9852E-04     8.3280E-04     F148       62

 BOND    =      344.9378  ANGLE   =      239.4549  DIHED      =      674.1530
 VDWAALS =      286.6490  EEL     =       12.6043  HBOND      =        0.0000
 1-4 VDW =      240.7275  1-4 EEL =     -135.7685  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1400       1.6628E+03     2.8716E-04     1.0005E-03     C138      305

 BOND    =      344.9385  ANGLE   =      239.4540  DIHED      =      674.1524
 VDWAALS =      286.6503  EEL     =       12.6043  HBOND      =        0.0000
 1-4 VDW =      240.7268  1-4 EEL =     -135.7685  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500       1.6628E+03     2.8027E-04     1.1236E-03     C28        70

 BOND    =      344.9381  ANGLE   =      239.4545  DIHED      =      674.1520
 VDWAALS =      286.6513  EEL     =       12.6043  HBOND      =        0.0000
 1-4 VDW =      240.7262  1-4 EEL =     -135.7686  RESTRAINT  =        0.0000


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1530       1.6628E+03     9.9432E-05     3.6438E-04     F148       62

 BOND    =      344.9387  ANGLE   =      239.4538  DIHED      =      674.1519
 VDWAALS =      286.6516  EEL     =       12.6044  HBOND      =        0.0000
 1-4 VDW =      240.7262  1-4 EEL =     -135.7686  RESTRAINT  =        0.0000

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|    Read coords time           0.00 ( 0.02% of Total)
|                Build the list             0.00 (92.60% of List )
|                Other                      0.00 ( 7.40% of List )
|             List time                  0.00 ( 0.08% of Nonbo)
|                   Short_ene time             2.43 (99.79% of Direc)
|                   Other                      0.01 ( 0.21% of Direc)
|                Direct Ewald time          2.43 (96.26% of Ewald)
|                Virial junk                0.00 ( 0.01% of Ewald)
|                Start synchronizatio       0.00 ( 0.01% of Ewald)
|                Other                      0.09 ( 3.72% of Ewald)
|             Ewald time                 2.53 (100.0% of Nonbo)
|             Other                      0.00 ( 0.01% of Nonbo)
|          Nonbond force              2.53 (73.33% of Force)
|          Bond/Angle/Dihedral        0.91 (26.50% of Force)
|          Other                      0.01 ( 0.17% of Force)
|       Force time                 3.45 (100.0% of Runmd)
|    Runmd Time                 3.45 (96.70% of Total)
|    Other                      0.12 ( 3.28% of Total)
| Total time                 3.57 (99.85% of ALL  )

| Number of list builds   :          1

| Highest rstack allocated:       1595
| Highest istack allocated:        319
|           Job began  at 21:34:24.075  on 03/15/2018
|           Setup done at 21:34:24.120  on 03/15/2018
|           Run   done at 21:34:27.652  on 03/15/2018
|     wallclock() was called   35225 times
